tests/test_helpers.py

```diff
@@ -0,0 +1,133 @@
+        """
+        Test to ensure False if returned  when no node is mapped to content object from node tree search
+        """
+        menu_contents = factories.MenuContentFactory()
+        child1 = factories.ChildMenuItemFactory(parent=menu_contents.root)
+        page_content = factories.PageContentWithVersionFactory(
+            language=self.language, version__created_by=self.get_superuser()
+        )
+        child2 = factories.ChildMenuItemFactory(parent=menu_contents.root)
+
+        result = search_content_object_from_node_tree(menu_contents, page_content)
+
+        self.assertFalse(result)
+
+        grandchild = factories.ChildMenuItemFactory(parent=child1)
+        grandchild1 = factories.ChildMenuItemFactory(parent=grandchild)
+        grandchild2 = factories.ChildMenuItemFactory(parent=grandchild1)
+
+        result = search_content_object_from_node_tree(menu_contents, page_content)
+
+        self.assertFalse(result)
+
+    def test_content_object_mapped_to_more_than_node(self):
+        """
+        Test to ensure first node found returned when more than one node is mapped to content object from node tree search
+        """
+        menu_contents = factories.MenuContentFactory()
+        child1 = factories.ChildMenuItemFactory(parent=menu_contents.root)
+        page_content = factories.PageContentWithVersionFactory(
+            language=self.language, version__created_by=self.get_superuser()
+        )
+        child2 = factories.ChildMenuItemFactory(parent=menu_contents.root)
+
+        grandchild = factories.ChildMenuItemFactory(parent=child1, content=page_content)
+        grandchild1 = factories.ChildMenuItemFactory(parent=grandchild)
+        grandchild2 = factories.ChildMenuItemFactory(parent=grandchild1, content=page_content)
+
+        result = search_content_object_from_node_tree(menu_contents, page_content)
+
+        self.assertEqual(result, grandchild)
+
+    def test_to_search_pollcontent_in_node_tree(self):
+        """
+        Test to ensure correct node mapped to poll content is returned from node tree search
+        """
+        poll = Poll.objects.create(name="Test poll")
+
+        poll_content = PollContent.objects.create(
+            poll=poll, language="en", text="example"
+        )
+        menu_contents = factories.MenuContentFactory()
+        child1 = factories.ChildMenuItemFactory(parent=menu_contents.root)
+        child2 = factories.ChildMenuItemFactory(parent=menu_contents.root, content=poll_content)
+
+        result = search_content_object_from_node_tree(menu_contents, poll_content)
+
+        self.assertEqual(result, child2)
+
+    def test_to_search_content_object_with_nodes_mapped_to_diff_content_objects_in_node_tree(self):
+        """
+        Test to ensure correct node mapped content object is returned with different nodes mapp[efrom node tree search
+        """
+
+        poll = Poll.objects.create(name="Test poll")
+
```

Comment:
This line break can be removed so that all of the assignments are grouped "AAA principle"